python/py-ezdxf/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup python        1.0
+PortGroup github        1.0
+
+name                    py-ezdxf
+version                 1.2.0
+revision                0
+
+github.setup            mozman ezdxf ${version} v
```

Comment:
the package can be fetched from PyPI which is preferred for Python packages, so please use that instead of GitHub

Reply:
Done